server/routes/handlers/book/helpers/validatePolice.test.ts

```diff
@@ -1,17 +1,32 @@
 import { validatePolice } from './validatePolice'
+import * as referenceDataExports from '../../../../referenceData'
 
 describe('validatePolice', () => {
-  it('returns valuesToSave and no errors if prison is submitted', () => {
+  beforeEach(() => {
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
You can cast this as jest.Mock, to avoid have to ignore it

Reply:
I'm thinking to keep as is here: this is consistent with e.g. `validateSentenceDetails.test.ts` and the example `jest.Mock` syntax I've tried here is rejected.

Reply:
ok; we can always take a look later